src/components/Form/Form.stories.tsx

```diff
@@ -72,6 +72,7 @@ export const Playground = (): ReactElement => {
 	const required = boolean("Required", true);
 	const disabled = boolean("Disabled", false);
 	const showSections = boolean("Show sections", false);
+	const showMultipleSections = boolean("Show multiple sections", true);

```

Comment:
Suggestion: We could get rid of this knob and make the "showSections" a select knob with 0, 1, 2, or 3 as options, if 0 then it's like we didn't have options.